release-tools/prow.sh

```diff
@@ -291,72 +212,24 @@ tests_need_alpha_cluster () {
 
 # Regex for non-alpha, feature-tagged tests that should be run.
 #
-# Starting with 1.17, snapshots is beta, but the E2E tests still have the
-# [Feature:] tag. They need to be explicitly enabled.
-configvar CSI_PROW_E2E_FOCUS_1_15 '^' "non-alpha, feature-tagged tests for Kubernetes = 1.15" # no tests to run, match nothing
-configvar CSI_PROW_E2E_FOCUS_1_16 '^' "non-alpha, feature-tagged tests for Kubernetes = 1.16" # no tests to run, match nothing
-configvar CSI_PROW_E2E_FOCUS_LATEST '\[Feature:VolumeSnapshotDataSource\]' "non-alpha, feature-tagged tests for Kubernetes >= 1.17"
-configvar CSI_PROW_E2E_FOCUS "$(get_versioned_variable CSI_PROW_E2E_FOCUS "${csi_prow_kubernetes_version_suffix}")" "non-alpha, feature-tagged tests"
+configvar COSI_PROW_E2E_FOCUS "$(get_versioned_variable COSI_PROW_E2E_FOCUS "${cosi_prow_kubernetes_version_suffix}")" "non-alpha, feature-tagged tests"
```

Comment:
I'm not sure if the tag is relevant for COSI.  For CSI tests, tests tagged with feature gates are only run in clusters that support them.  We can keep them for now and remove them if we are sure they are not needed.

Reply:
Fixed.